<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowRecherche</class>
 <widget class="QWidget" name="WindowRecherche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="l_typeDeRecherche">
       <property name="text">
        <string>Type de recherche:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_donnee"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_datesFiltre">
     <item>
      <layout class="QVBoxLayout" name="layout_ADate">
       <item>
        <widget class="QCheckBox" name="check_avant">
         <property name="text">
          <string>Avant</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="de_avant"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layout_ApDate">
       <item>
        <widget class="QCheckBox" name="check_apres">
         <property name="text">
          <string>Apr√®s</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="de_apres"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="check_trier">
     <property name="text">
      <string>Trier</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tw_donnee"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_fermeture">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_fermer">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
